### PR TITLE
chore: simplify color warnings and other fixes for themebuilder

### LINKED ISSUE #2323: Improve color warnings for the themebuilder
Improve the warning system for the colors and create documentation on how to use the tool.

* Simplify the color warning system to only show 3:1 warning against background colors
   * Removed the color warning that said that the contrast color did not have good enough contrast against all the base colors. This has been fixed in the tool.
* Add documentation page that explains how to use the tool and how to pick the colors

Can be tested here: https://pr-2333.theme.designsystemet.no/

### PR DESCRIPTION
**Pull request by Thunear**
resolves #2323

Also some other fixes:
* Add footer
* Fix the header and text inside the color modal.
* Styling fixes on the component preview
* Hide contrast and border radius selects until these are functional in both code and figma

### CODE REVIEW

File: apps/theme/app/slik-bruker-du-verktoyet/page.tsx
Comment (Barsnes) [inline apps/theme/app/slik-bruker-du-verktoyet/page.tsx:1]: Burde me ha ein tittel med "Hva er et tema?", der me seier kva som inngår?

File: apps/theme/public/img/1080p.jpg
Comment (Barsnes) [inline apps/theme/public/img/1080p.jpg:1]: I can't see this being used anywhere

File: apps/theme/public/img/test.png
Comment (Barsnes) [inline apps/theme/public/img/test.png:1]: I can't see this being used anywhere

File: apps/theme/public/img/test2.png
Comment (Barsnes) [inline apps/theme/public/img/test2.png:1]: I can't see this being used anywhere

**Review by Barsnes — COMMENTED** [review]
https://github.com/user-attachments/assets/3058a10c-3cd4-44e4-977d-fb04c6ee24db
Can we add a min-height of 100vh?

File: apps/theme/app/slik-bruker-du-verktoyet/page.tsx
Comment (Thunear) [reply apps/theme/app/slik-bruker-du-verktoyet/page.tsx:1]: Ja, enig. La til ein liten sekjon som gir eit raskt overblikk.

**Review by Barsnes — APPROVED** [review]

**Pull request merged**